src/Builders/CreditnoteBuilder.php

```
@@ -8,4 +8,22 @@ class CreditnoteBuilder extends Builder
     protected $model = Creditnote::class;
+
+
+
+
+    /**
+     * @param $id
+     * @param $timestamp
+     *
+     * @return mixed
+     */
+    public function book($id, $timestamp)
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)